systemtests/src/main/java/io/enmasse/systemtest/scale/ScaleTestClient.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2019, EnMasse authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.enmasse.systemtest.scale;
+
+import io.enmasse.systemtest.model.address.AddressType;
+import io.enmasse.systemtest.scale.metrics.ScaleTestClientMetricsClient;
+
+public class ScaleTestClient <X extends ScaleTestClientMetricsClient>{
```

Comment:
I think the convention is to use T as the first generic parameter, we should stick to that.